### PR TITLE
WIP: Help replacement

### PR DESCRIPTION
**Pull request by jburel**
Migrate help

### CODE REVIEW

File: explore/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@ <h2 class="header-red">OME for Scientists</h2>
         <div class="row">
             <img class="medium-3 medium-offset-1 columns" alt="OMERO for Scientists" src="{{ site.baseurl }}/img/icons/user-groups_scientists.svg">
             <div class="medium-8 columns">
-                <p>Sign up for our <a href="https://help.openmicroscopy.org/demo-server.html">demo server</a> to test out the functionality with your own data.</p>
+                <p>Sign up for our demo server to test out the functionality with your own data.</p>
```
Comment (mtbc) [inline explore/index.html:27]: Perhaps link to https://demo.openmicroscopy.org/signup/.

File: omero/features/analyze/index.html [hunk 2]
```diff
@@ -108,8 +108,30 @@ <h4>Tabular Data support</h4>
...
+                <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_orbit.svg">
+                <h4>QuPath</h4>
+                <p>Quantitative Pathology and Bioimage Analysis that connects to OMERO.</p>
+                <a class="button hollow tiny" href="https://omero-guides.readthedocs.io/en/latest/qupath/docs/index.html" target="_blank">View guide</a>
+                <a class="button hollow tiny" href="https://qupath.github.io/" target="_blank">Find out more</a>
+            </div>
+            <div class="omero-feature-container tall-content column text-center">
+                <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_orbit.svg">
+                <h4>CellProfiler</h4>
+                <p>quantitative analysis of biological images that connects to OMERO.</p>
```
Comment (mtbc) [inline omero/features/analyze/index.html:124]:
```suggestion
                <p>Quantitative analysis of biological images that connects to OMERO.</p>
```

File: explore/index.html (hunk 1 above)
Comment (jburel) [reply explore/index.html:27]: Good call.

**Comment by pwalczysko** [comment]
I think 

Screen Shot 2020-07-27 at 14 52 37


should point to https://docs.openmicroscopy.org/omero/5.6.2/sysadmins/import-scenarios.html#parallel-import

**Comment by pwalczysko** [comment]
Instead of https://github.com/ome/www.openmicroscopy.org/pull/396/files#diff-20494d8d48cbfaa98988087e56275885R36 I would go for link to https://omero-guides.readthedocs.io/projects/introduction/en/latest/search-omero.html?highlight=search and https://omero-guides.readthedocs.io/projects/introduction/en/latest/annotate.html

**Comment by pwalczysko** [comment]
To the "Browse and view",  suggest to add one more link, for Browsing, https://omero-guides.readthedocs.io/en/latest/introduction/docs/data-management.html#browsing-and-rendering

Screen Shot 2020-07-27 at 15 12 37

**Comment by pwalczysko** [comment]
Screen Shot 2020-07-27 at 15 15 44

Swap the order of buttons here - first, the less scary omero-guide, then the full permissions overview.

**Comment by pwalczysko** [comment]
Screen Shot 2020-07-27 at 15 17 24
Swap the order of buttons here. First, the developer guide (less complex), then the model explanation (hard to navigate and realize what to look for)

**Comment by pwalczysko** [comment]
The analyse part of OMERO features is not right in the first box, see screenshot below.

Screen Shot 2020-07-27 at 16 51 59

The first link is pointing to omero-guides with the 3rd party tools. This is fine, but because it is out of place and mislabelled, then it is underselling that work.

The second link is pointing to help, which we wanted to get rid of.

Suggest:
1. Create a new icon with something like "use 3rd party tools to analyze your data in OMERO" (yes, there is a section below on the same page which has this, but this is hardly visible, as you have to scroll down to even see it.
2. Put this icon also under "New" (features), maybe replacing ROI folders ?
3. Have this icon with "analytical results attachment" too, but with one link pointing to https://omero-guides.readthedocs.io/projects/introduction/en/latest/annotate.html (the text there needs to be improved) and the other link pointing to https://omero-guides.readthedocs.io/projects/upload/en/latest/metadata.html

**Pull request closed without merging**